### PR TITLE
Contribution change between two entities

### PR DESCRIPTION
**Pull request by wei-enwang**
A new page is created to examine the contribution value change when feature values change. Currently, the page only allows users to select entities from the dataset. Therefore, the next step might be to add API to allow users to create customized data and see what the feature contributions for their data will be.

### CODE REVIEW

**Review by zyteka — CHANGES_REQUESTED** [review]
Great job with this! This will be a really helpful page, especially once we factor in the concept of time

File: sibylapp/view/entity_difference.py [hunk 1]
```diff
@@ -0,0 +1,213 @@
...
+    st.metric(
+        "{prediction} Difference between {entity} #2 and {entity} #1:".format(
+            prediction=get_term("Prediction"), entity=get_term("Entity")
+        ),
+        pred_format_func(difference),
+    )
+
+
+def show_sorted_contributions(to_show, sort_by):
+    show_legend()
```
[the remark this answers is not in the document]
Comment (wei-enwang) [reply sibylapp/view/entity_difference.py:48]: This is what I set for right now. What do you think?
Screen Shot 2023-07-26 at 5 32 11 PM
Here's what I wrote in the sidebar.
Screen Shot 2023-07-26 at 5 30 11 PM

File: .gitignore [hunk 2]
```diff
@@ -119,6 +119,7 @@ venv/
 ENV/
 env.bak/
 venv.bak/
+.vscode/
```
Comment (zyteka) [inline .gitignore:122]: Nitpick but can you move this down below the `#pycharm` comment, with a comment `vscode`?

File: pages/4_Compare_Cases.py [hunk 3]
```diff
@@ -0,0 +1,30 @@
+import streamlit as st
```
Comment (zyteka) [inline pages/4_Compare_Cases.py:1]: Thinking about this a bit more, can you move this page to before "Understand the model"? That way the first three pages have local explanations, and the last page has a global explanation

**Review by zyteka — APPROVED** [review]
Looks great! Just two little things and then please go ahead and merge

**Pull request merged**